### PR TITLE
Week 5 cjpriedel

### PR DESCRIPTION
**Pull request by cjpriedel**

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -15,42 +15,154 @@
...
+    let stringWithA = str.replace(/a/gi, "4")
+    let stringWithE = stringWithA.replace(/e/gi, "3")
+    let stringWithI = stringWithE.replace(/i/gi, "1")
+    let stringWithO = stringWithI.replace(/o/gi, "0")
+    return stringWithO
+  }
+
+//   PASS  ./code-challenges.test.js
+//   l33tSp34k
+//     √ Create a function that takes in a string and returns a coded message (3 ms)
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:60]: Very cool

File: code-challenges.test.js [hunk 2]
```diff
@@ -15,42 +15,154 @@
...
+// // Expected output: ["Apple", "Banana", "Orange"]
+// // Expected output: ["Cherry", "Blueberry", "Peach"]
+
+// // b) Create the function that makes the test pass.
+
+const returnItemsOfLetter = (arr, letter) => {
+    return arr.filter(val => {
+      return val.includes(letter) || val.includes(letter.toUpperCase())
+    })
+  }
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:98]: Good work with edge cases

File: code-challenges.test.js [hunk 3]
```diff
@@ -15,42 +15,154 @@
...
+
+const fullHouse = (arr) => {
+    if (arr[0] === arr[1,2] && arr[3] === arr[4] || 
+        arr[0] === arr[2,3] && arr[1] === arr[4] ||
+        arr[0] === arr[3,4] && arr[1] === arr[2])
+        return true
+    else{
+        return false
+    }
+ }
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:148]: Solid!

File: interview-questions.md [hunk 4]
```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 4. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: Relational databases are stored in data tables.
 
-  Researched answer:
+  Researched answer: Relational databases are similar to the way info can be stored in Microsoft Excel, using columns and rows with labels to identify data by using all tables that are connected to each other. Non-relational databases can perform slower than their counterpart, and anything can be stored as a value, as long as a key or name is assigned to it.
```
Comment (WalkerTexas-coder) [inline interview-questions.md:36]: Great follow up!

File: ruby-challenges.rb [hunk 5]
```diff
@@ -19,17 +25,51 @@
...
+    def pedal_faster(num1)
+        @current_speed += num1
+    end
+    def brake(num2)
+        if @current_speed - num2 <=0
+        @current_speed =0
+        else
+            @current_speed -= num2
+    end
+  end
```
Comment (WalkerTexas-coder) [inline ruby-challenges.rb:56]: Nice!

File: code-challenges.test.js (hunk 2 above)
Comment (WalkerTexas-coder) [inline code-challenges.test.js:98]: Good work with edge cases

**Review by WalkerTexas-coder — APPROVED** [review]
Great work CJ! This looks really good! I want to see you push yourself next week and document some creative refactoring and see what it yields for you!
